csharp/CMakeLists.txt

```diff
@@ -48,7 +48,8 @@ endif()
 
 # The Release/Debug can be stripped probably
 # Also probably don't need to use multiple `<TargetFrameworks>` which is the reason there's an extra netstandard2.0/ level
-set(OPENSTUDIO_CSHARP_DLL "${CSHARP_LIBRARY_OUTPUT_DIRECTORY}/${CMAKE_BUILD_TYPE}/netstandard2.0/OpenStudio.dll")
+# Note: can't use CMAKE_BUILD_TYPE as it's empty for multi generators (MSVC for eg). So we use a genex
+set(OPENSTUDIO_CSHARP_DLL "${CSHARP_LIBRARY_OUTPUT_DIRECTORY}/$<CONFIG>/netstandard2.0/OpenStudio.dll")
```

Comment:
See the note.

Reply:
I replaced the CMAKE_BUILD_TYPE with a genex too. This was working on CI because we do pass explicitly `cmake -DCMAKE_BUILD_TYPE:STRING=Release` when building on MSVC. But MSVC by default is **Multi-configuration** and CMAKE_BUILD_TYPE is empty if used like this.